drake/systems/LCMSystem.cpp

```diff
@@ -0,0 +1,55 @@
+
+#include <iostream>
+#include "LCMSystem.h"
+
+#if defined(_WIN32) || defined(_WIN64)
```

Comment:
If _WIN64 is defined, then _WIN32 is also defined. See https://msdn.microsoft.com/en-us/library/b0084kay.aspx.